qiwis.py

```diff
@@ -37,7 +37,6 @@
 
 logger = logging.getLogger(__name__)
 
-

```

Comment:
Please discard this change since it follows the rule (2 empty lines between global definitions).

Reply:
Please discard this change.